CafeTime/CafeTime/Extensions.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  Extensions.swift
+//  CafeTime
+//
+//  Created by Vladysalv Vyshnevksyy on 1/27/17.
+//  Copyright © 2017 Vladysalv Vyshnevksyy. All rights reserved.
+//
+
+import UIKit
+import M13Checkbox
+
+extension UIView {
```

Comment:
Better make a separate file called "UIView+Extension" and place it here
and keep separate extension for each object. not all in one place